kotlin/CHANGELOG.md

```diff
@@ -1,6 +1,22 @@
 Change Log
 ==========
 
+## Version 0.8.0
+
+_2019-3-12_
+
+ * Breaking change, `ViewBuilder` is now `ViewBinder`. Adds `buildScene()`
+   method. (#57)
+
```

Comment:
Nit: Did you mean to leave these blank lines? None of the other entries have them.